### ISSUE TITLE
Error while symlinking

### ISSUE THREAD

**Issue by gVallverdu** [andrewrynhard/atom-latex-plus#48]
Hello,

Atom 1.7.2
latex-plus 0.8.7
OS : Mac OS X 10.11.4

I got an error while symlinking after compilation : 

![latexplus-error](https://cloud.githubusercontent.com/assets/5417018/14783734/f34246c2-0af1-11e6-8c3f-2a8bc5f754aa.png)

Thank you for your help

**Comment by svkurowski** [comment]
Hey, can you show us the output of

```
ls -l /Users/gvallver/Recherche/Présentations/IPREM/Axe3/280416_vallverdu.pdf
ls -al /Users/gvallver/Recherche/Présentations/IPREM/Axe3/.latex
```

?
(You might have to fix the paths in the commands incase I might typos or due to the special character)

**Comment by gVallverdu** [comment]
Hello
Here it is : 

```
ls -l /Users/gvallver/Recherche/Présentations/IPREM/Axe3/280416_vallverdu.pdf
-rw-r--r--@ 1 gvallver  staff  1457294 25 avr 17:16 /Users/gvallver/Recherche/Présentations/IPREM/Axe3/280416_vallverdu.pdf
```

```
ls -al /Users/gvallver/Recherche/Présentations/IPREM/Axe3/.latex
total 3192
drwxr-xr-x  12 gvallver  staff      408 26 avr 09:31 .
drwxr-xr-x  10 gvallver  staff      340 26 avr 09:31 ..
-rw-r--r--   1 gvallver  staff     1816 26 avr 09:31 280416_vallverdu.aux
-rw-r--r--   1 gvallver  staff    34281 26 avr 09:31 280416_vallverdu.fdb_latexmk
-rw-r--r--   1 gvallver  staff    40624 26 avr 09:31 280416_vallverdu.fls
-rw-r--r--   1 gvallver  staff    74845 26 avr 09:31 280416_vallverdu.log
-rw-r--r--   1 gvallver  staff      610 26 avr 09:31 280416_vallverdu.nav
-rw-r--r--   1 gvallver  staff        0 26 avr 09:31 280416_vallverdu.out
-rw-r--r--   1 gvallver  staff  1457294 26 avr 09:31 280416_vallverdu.pdf
-rw-r--r--   1 gvallver  staff        0 26 avr 09:31 280416_vallverdu.snm
-rw-r--r--   1 gvallver  staff     7257 26 avr 09:31 280416_vallverdu.synctex.gz
-rw-r--r--   1 gvallver  staff       66 26 avr 09:31 280416_vallverdu.toc
```

My first assumption was about accents in the paths. But I did a try in a path without accents and I got the same error.

**Comment by Edenharder** [comment]
Yeah, I have the same error while symlinking. 

Atom 1.7.3
latex-plus 0.8.7
OS : Mac OS X 10.11.4

**Comment by svkurowski** [comment]
Okay, so apparently you already have a file named `280416_vallverdu.pdf` in the root folder of your project. This package creates the pdf and synctex files in a hidden subfolder (`.latex` by default, but it's configurable) and then symlinks those files into the root folder. To use the package you need to rename the pdf from the root folder (or remove it, but renaming is more safe) and also the file named  `280416_vallverdu.synctex.gz`. Probably this pdf was created by you running `pdflatex` before using this package here.

I think we need a much nicer error message for this though! We can't just remove the files from the package as they might be important to the user and he might lose data. An alert dialog asking whether the files should be removed might help.

**Comment by andrewrynhard** [comment]
@svkurowski I agree. In #49 we are discussing wether we should symlink or not. The idea behind the .latex folder was so that all the output could be put into `.gitignore` with a simple `.latex` entry. Only two items really make sense to have in the root. The first being the PDF and the second being the synctex file. In addition to a nice error message perhaps we should copy the file to the root rather than symlink?

**Comment by svkurowski** [comment]
Well we would have the same problem when copying. If there already is a file at the path, the user has to perform an action.

Also, I just added my personal opinion to the issue you linked.

**Comment by andrewrynhard** [comment]
Yes. We would still have to handle the case when a file is already there. Copying is a completely unrelated issue that I should probably open another issue for. Reason is that for one of my projects I actually keep a copy of the PDF in my repository and reference it from a website. With symlinks I do not get this option as I ignore the .latex directory.

**Comment by andrewrynhard** [comment]
Closed by #71
